src/test/acceptance/features/add-children-dob.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Add children's date of birth
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to enter the dates of birth of my children who are 3 years old or younger
+
+  Background:
+    Given I have entered my details up to the add your childrens dates of birth page
+
+  Scenario: Enter one child's details
+    When I enter the details of my child who is under 3
+    And I click continue
+    Then I am shown the are you pregnant page
+
+  Scenario: Enter two children's details
+    When I enter the details of my two children who are under 3
```

Comment:
Should we say under 4 as that's the required age.

Reply:
Yes, my mistake, I had the phrase 3 or younger in my head and just did this wrongly.